x-pack/libbeat/common/aws/credentials.go

```diff
@@ -45,6 +46,16 @@ func GetAWSCredentials(config ConfigAWS) (awssdk.Config, error) {
 	// Please see https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html
 	// with more details.
 	if config.ProfileName != "" {
+		if config.SharedCredentialFile != "" {
```

Comment:
I think it should be possible to set the credential file even if the profile name is not set, one may want to use the default profile in a credential file.